src/uwtools/tests/test_cli.py

```diff
@@ -289,6 +289,57 @@ def test__dispatch_forecast_run():
     m().run.assert_called_once_with(cycle="2023-01-01T00:00:00")
 
 
+@pytest.mark.parametrize(
+    "params",
+    [
+        (STR.realize, "_dispatch_rocoto_realize"),
+        (STR.validate, "_dispatch_rocoto_validate"),
+    ],
+)
+def test__dispatch_rocoto(params):
+    submode, funcname = params
+    args = ns()
+    vars(args).update({STR.submode: submode})
+    with patch.object(cli, funcname) as m:
```

Comment:
What does `m` stand for here? Placeholder? Not sure if I know the background

Reply:
yeah these `m` calls are patching in placeholder modules - but our book would probably tell us to stop trying to save bytes and just call them `module` haha. I'll see if I can track all of these down and change them

Reply:
Looks like they were just here, so quick fix!